bluepyopt/tests/test_ephys/test_create_acc.py

```diff
@@ -0,0 +1,101 @@
+"""Tests for create_acc.py"""
+
+# pylint: disable=W0212
+
+import os
+import re
+import json
+
+from . import utils
+from bluepyopt.ephys import create_acc
+
+
+import pytest
+
+DEFAULT_ARBOR_REGION_ORDER = [
+    ('apic', 4),
+    ('axon', 2),
+    ('dend', 3),
+    ('soma', 1)]
+
+
+@pytest.mark.unit
+def test_create_acc():
```

Comment:
Could you commit an expected resulting arbor code file in the repo, and compare with what comes out of create_hoc

Reply:
Ok, so the output will (model-dependent) look like the `simple_cell.json`, `label_dict.acc` and `decor.acc` in https://github.com/BlueBrain/BluePyOpt/pull/393#issue-1196232421. The tests for `create_acc` as implemented now follow 1:1 the `create_hoc` tests. 

Would you like both of these tests to check the `create_...` output instead against reference files (separately for hoc and JSON/ACC)? I'm currently not sure what it would add over checking with assertions as done now. 

If your request is to test that `create_hoc`'s output is consistent with that of `create_acc` by parsing both outputs - which specific properties would you like to check? The `hoc`-inspection in `test_create_hoc` is rather terse at the moment.

Reply:
Well, it would just be good to have a reference acc file, to test the create_acc functionality against. I don't see any such file in the PR. Maybe I missed it (or is it maybe stored in a string?) ?

Reply:
Sure, I've added the reference files in e88168f in the `testdata/acc/CCell` folder and am comparing the cell JSON and the label_dict cable component values. For the decor, we'll have to stick to string comparison as Arbor's S-expression parser is not exposed and the decor object does not give access to internal data.

Reply:
Both could be changed, the latter is simpler.